.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: node_js
-node_js: 12
-script:
-- npm run lint
-- npm run test
-- npm run build
-- npm run docs
-after_success:
-- npx semantic-release@17
-- codecov
-deploy:
```

Comment:
The frontend-platform build is a bit different than others in that it publishes the documentation site to Github Pages.  I'm not seeing this behavior preserved in the Github Actions version.

Here's an example of this being done via Github Actions in another repository: https://github.com/openedx/openedx-tech-radar/blob/main/.github/workflows/release.yml#L24

Reply:
Added the GitHub Pages deployment step.